### PR TITLE
# Change Dependencies for SCSS: now using "sass"

### PR DESCRIPTION
**Pull request by 3dyonic**
https://monday.monday.com/boards/245345663/pulses/2647369549
* Change "node-sass" to dart "sass" version 1.49.9
  * Version is the same as in the Da Pulse Monolith.

### CODE REVIEW

**Review by sahariko — COMMENTED** [review]
It seems like you didn't save `sass` to `devDependencies`, nor did you remove `node-sass` from them.

File: package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "clean-css-cli": "^4.3.0",
     "jest": "^27.5.1",
     "node-plop": "^0.26.2",
-    "node-sass": "^4.14.1",
+    "sass": "1.49.9",
```
Comment (sahariko) [inline package.json:48]: Let's use the latest version (`1.51.0` as of this message).

Comment (3dyonic) [reply package.json]: done

**Review by sahariko — APPROVED** [review]
Let's lose the pound sign (#) in the commit message

**Comment by 3dyonic** [comment]
> Let's lose the pound sign (#) in the commit message

why?
it's markdown.

**Comment by sahariko** [comment]
> why? it's markdown.

WDYM? This is not markdown, this is the commit message that will appear in this repo's commit list. It doesn't appear as markdown anywhere, and if it would, it would appear as an H1 title, which I don't think is what you want anyway.

**Pull request merged**